runtime/lib/runtime.cpp

```diff
@@ -245,4 +247,167 @@ void wait(Event event) {
   throw std::runtime_error("runtime is not enabled");
 }
 
+std::vector<float> Tensor::getData() {
```

Comment:
@nsmithtt im not feeling too good about this guy, any ideas on a more clean approach?

I can get Tensor struct back into callback golden function, but getting the shared_ptr<void> into some kind of tensor form back into python is tricky. Especially without any metadata about tensor data (ie volume) - which I had to add in.

Reply:
If we wanted to be fancy we could use pybind11 buffer protocol: https://pybind11.readthedocs.io/en/stable/advanced/pycpp/numpy.html

I'm OK with returning a float vector though.  I think we should move this function though, see comment in other thread ***.

Reply:
fixed